code/game/machinery/doors/airlock.dm

```diff
@@ -952,8 +952,8 @@ About the new airlock wires panel:
 			if(src.isWireCut(AIRLOCK_WIRE_DOOR_BOLTS))
 				to_chat(usr, SPAN_WARNING("The door bolt control wire is cut - Door bolts permanently dropped."))
 			else if(activate)
-				if(issilicon(usr) && !player_is_antag(usr.mind))
-					to_chat(usr, SPAN_WARNING("Your programming prevents you from lowering the door bolts."))
+				if(issilicon(usr) && density && !player_is_antag(usr.mind))
```

Comment:
Is the density check just meant to exclude drones? 

Does it affect AIs that are holograms or anything? You might wanna do a more exclusive istype(usr, drone) thing.

Reply:
the density thing checks whether the door is closed or not